frameworks/Scala/zio-http/zio-http.dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM alpine as builder
```

Comment:
You could use a fixed alpine version here to avoid that future updates can break the build